easybuild/easyconfigs/s/SearchGUI/SearchGUI-3.3.3-Java-1.8.0_152.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PackedBinary'
+
+name = 'SearchGUI'
+version = '3.3.3'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://github.com/compomics/searchgui'
+description = """SearchGUI is a user-friendly open-source graphical user interface for configuring and
+ running proteomics identification search engines and de novo sequencing algorithms,
+ currently supporting X! Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA,
+ Novor and DirecTag."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://genesis.ugent.be/maven2/eu/isas/searchgui/SearchGUI/%(version)s/']
+sources = ['SearchGUI-%(version)s-mac_and_linux.tar.gz']
+checksums = ['4589cf8582b4a56f6cddc79ff3bf6601067899046178834141288e3a79897f5b']
+
+dependencies = [
+    ('Java', '1.8.0_152'),
+    ('Mono', '4.6.2.7')
+]
+
+sanity_check_paths = {
+    'files': ['SearchGUI-%(version)s.jar'],
+    'dirs': ['lib', 'resources'],
+}
+
+modaliases = {
+    'SearchCLI': 'java -cp $EBROOTSEARCHGUI/SearchGUI-%(version)s.jar eu.isas.searchgui.cmd.SearchCLI',
+    'PathSettingsCLI': 'java -cp $EBROOTSEARCHGUI/SearchGUI-%(version)s.jar eu.isas.searchgui.cmd.PathSettingsCLI',
+    'FastaCLI': 'java -cp $EBROOTSEARCHGUI/SearchGUI-%(version)s.jar eu.isas.searchgui.cmd.FastaCLI'
+}
+
+# workaround fixing MyriMatch on Linux - error "locale::facet::_S_create_c_locale name not valid"
+modextravars = {'LC_ALL': 'C'}
```

Comment:
@korbini This looks like a site-specific fix to me, which means it shouldn't be included in the easyconfig...

How did you trigger the error (so we can test this ourselves to see if we can reproduce it without setting `$LC_ALL` to `C`)?